index.html

```diff
@@ -85,6 +85,268 @@ <h1>
+                        <li class="dot"><img src="/images/Dot.png" alt=""></li>
+                        <li class="year">2015</li>
+                      </ul>
+                    </div>
+                    <div>
+                      <p>A daily selection of privately<br>
+                        personalized reads; no accounts or<br>
+                        sign-ups required.
+                     </p>
+                    </div>
+                    <div>
+                      <a href="html" class="btn">HTML</a>
+                    <a href="css" class="btn">CSS</a>
+                    <a href="JavaScript" class="btn">JavaScript</a>
+                    </div>
+                    <div class="btn_see">
+                      <a href="#">See Project</a>
+                    </div>
+                </div>
+              </article>
+            </div>
+          </section>
+        </div>
+
+        <div class="about_section">
+          <section class="about_me">
+            <div class="about_info">
+              <div class="about_me_title">
+                <h3>About<br>
+                  Myself</h3>
+              </div>
+              <div class="about_desc">
+                <p>
+                  Hello I’m a software developer! I can help<br>
+                   you build a product , feature or website <br>
+                   Look through some of my work and<br>
+                    experience! If you like what you see and<br>
+                     have a project you need coded, don’t<br>
+                      hestiate to contact me.
+                </p>
+              </div>
+              <div class="let_connect">
+                <a href="#">
+                  <h4>LET’S CONNECT</h4>
+                </a>
+              </div>
+                <div class="social-images">
+                  <ul>
+                    <li>
+                      <a href="#">
+                        <img src="images/github.png" alt="github-icon" />
+                      </a>
+                    </li>
+                    <li>
+                      <a href="#">
+                        <img src="images/Linkedinicon.svg" alt="Linkedin-icon" />
+                      </a>
+                    </li>
+                    <li>
+                      <a href="#">
+                        <img src="images/twitter.png" alt="twitter-icon" />
+                      </a>
+                    </li>
+                    <li>
+                      <a href="#">
+                        <img src="images/Vector(1).svg" alt="line-icon" />
+                      </a>
+                    </li>
+                    <li>
+                      <a href="#">
+                        <img src="images/Vector(2).svg" alt="socail-icon" />
+                      </a>
+                    </li>
+                  </ul>
+              </div>
+              <div class="btn_resume">
+                <a href="#">Get My Resume</a>
+              </div>
```

Comment:
- [ ] [Optional]  Kindly follow the button styling: Active, Hover and Pressed (see design)
### Design in Figma
![figma](https://user-images.githubusercontent.com/81727988/170819584-df1e93d6-aecf-4d53-9d25-3cbcb41396b7.png)